.github/workflows/test.yaml

```diff
@@ -40,6 +40,7 @@ jobs:
       - name: Run test
         run: |
           go-acc -o cover.out ./... -- \
+          -bench=. \
```

Comment:
This should be switched by https://github.com/pion/stun/blob/master/.github/.ci.conf